lib/shared/thread-utils.js

```diff
@@ -52,13 +52,33 @@ function viewerIsMember(threadInfo: ?(ThreadInfo | RawThreadInfo)) {
   );
 }
 
+function threadIsInHome(threadInfo: ?(ThreadInfo | RawThreadInfo)) {
+  return threadInfo && threadInfo.currentUser.subscription.home;
+}
+
 function threadInChatList(threadInfo: ?(ThreadInfo | RawThreadInfo)) {
   return (
     viewerIsMember(threadInfo) &&
     threadHasPermission(threadInfo, threadPermissions.VISIBLE)
   );
 }
 
+function threadInBackgroundChatList(threadInfo: ?(ThreadInfo | RawThreadInfo)) {
+  return (
+    viewerIsMember(threadInfo) &&
+    threadHasPermission(threadInfo, threadPermissions.VISIBLE) &&
+    !threadIsInHome(threadInfo)
+  );
+}
+
+function threadInHomeChatList(threadInfo: ?(ThreadInfo | RawThreadInfo)) {
+  return (
+    viewerIsMember(threadInfo) &&
```

Comment:
Same here, let's just call into `threadInChatList(threadInfo)` for these first two conditions